tests/Plivo.Test/Resources/TestMessage.cs

```diff
@@ -1,16 +1,15 @@
 using System.Collections.Generic;
-using NUnit.Framework;
+using Xunit;
```

Comment:
if your using Xunit, please use it in all places. Not good to keep two framework

Reply:
Fixed